.github/workflows/test_latest_versions.yml

```diff
@@ -20,12 +20,12 @@ jobs:
       max-parallel: 4
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        python-version: ['3.10', '3.11', '3.12', '3.13']
         include:
           - os: macos-latest
             python-version: 3.11
-          - os: windows-latest
-            python-version: 3.11
+          # - os: windows-latest
+          #   python-version: 3.11
```

Comment:
#54 should allow these tests to run properly, let's remember to remove the comment